pysph/tools/ipy_viewer.py

```diff
@@ -656,8 +676,26 @@ def _save_figure_handler(self, change):
         self._widgets.save_figure.value = ""
 
     def _delay_box_handler(self, change):
+
         self._widgets.play_button.interval = change['new']*1000
 
+    def _save_all_plots_handler(self, change):
+
+        if change['new']:
+            self._widgets.save_figure.disabled = True
+        elif not change['new']:
+            self._widgets.save_figure.disabled = False
+
+    def _save_plot(self):
+
+        abs_path = os.path.abspath(self.path)
+        if self._widgets.save_all_plots.value:
+            mkdir('/all_plots')
+            self.figure.savefig(
+                'frame_%s.png' % self._widgets.frame.value,
```

Comment:
This should be `all_plots/frame...` right?

Reply:
I've updated this method in the new commit, it no longer uses this function, and yes, it was supposed to be `all_plots/frame..`.